tests/test_e2e.py

```diff
@@ -13,6 +13,21 @@
 logging.basicConfig(level=logging.INFO)
 
 
+def _get_app_and_auth_code(
+        client_id,
+        client_secret=None,
+        authority="https://login.microsoftonline.com/common",
+        port=44331,
+        scopes=["https://graph.windows.net/.default"],
```

Comment:
We try to move everything from AAD Graph to Microsoft Graph.
Would it make sense to use https://graph.microsoft.com/.default instead?